crates/cxx-qt-lib/build.rs

```diff
@@ -4,10 +4,14 @@
 // SPDX-License-Identifier: MIT OR Apache-2.0
 
 fn main() {
-    let qt_modules = vec!["Core", "Gui"]
-        .iter()
-        .map(|m| String::from(*m))
-        .collect();
+    let qt_modules = vec![
+        "Core",
+        #[cfg(feature = "qt_gui")]
```

Comment:
This might need to use env vars instead? https://doc.rust-lang.org/cargo/reference/features.html#build-scripts

Reply:
Done